scoping.md

```diff
@@ -127,11 +161,14 @@ which may or may not be desirable.
   without exposing it to the rest of the system**, reducing exposure
   to sensitive data and providing greater control and transparency for users.
 
-The use of "encrypted" is again seen as problematic.  In addition,
-the text is at best ambiguous about the scope, as the text about
-tools is constrained to "TEE" development, but the aims are broader,
-applying to protecting "data in use", where all of Figure 1 would
-claim to be about protecting data in use.
+The TAC observed the following issues with the above text:
+
+1. The use of "encrypted" is again seen as problematic.
```

Comment:
We should state why... encrypted refers to a mechanism rather than a property we are seeking. You can protect by access control and not actually require encryption.

Reply:
Expanded text in lines 150-153.

Reply:
It was in 150-151, but I've now expanded it to include your text.